02-peer-protocol.md

```diff
@@ -378,10 +378,14 @@ This message introduces the `channel_id` to identify the channel. It's derived f
 #### Requirements
 
 Both peers:
-  - if `option_static_remotekey` or `option_anchor_outputs` was negotiated:
-    - `option_static_remotekey` or `option_anchor_outputs` applies to all commitment transactions
+  - if `option_static_remotekey`, `option_anchor_outputs` or
```

Comment:
since `option_anchors` has been added to cover `_outputs` and `_zero_fee_htlc_tx`, why is it not used here?

Reply:
Worse, you don't define what the effect is if *both* are negotiated!

```
- if `option_anchors_zero_fee_htlc_tx` was negotiated:
  - `option_anchor_outputs` is ignored.
...
Rationale:
`option_anchors_zero_fee_htlc_tx` is considered superior to `option_anchor_outputs`, so is favored if both are negotiated.
```

Reply:
If both are negotiated, then both can apply which I don't think is a problem since the latter is a superset of features.

 I added a rationale and also specified `option_anchors` the places where `option_static_remotekey` defines behavior (since technically `option_anchors_zero_fee_htlc_tx` does no longer depend on `option_static_remotekey`): https://github.com/lightningnetwork/lightning-rfc/pull/824/commits/75a4cde905ad614307b6573ab14109a41bce5a83